Gopkg.lock

```diff
@@ -794,6 +800,15 @@
   packages = ["ssh/terminal"]
   revision = "a49355c7e3f8fe157a85be2f77e6e269a0f89602"
 
+[[projects]]
+  branch = "master"
+  name = "golang.org/x/mobile"
+  packages = [
+    "asset",
+    "internal/mobileinit"
+  ]
+  revision = "507816974b79c76a5fe70f9580265cd57dc78bbe"
+
```

Comment:
@DataDog/agent-core 

`inv deps` keeps wanting to add this to the `Gopkg.lock`. Should I just remove it?